### PR TITLE
Fixed archetype Basic project build failure

### PR DESCRIPTION
**Pull request by AbhayChandel**
The problem has been known earlier: Maven profiles issue with Java 11(https://github.com/devonfw/devon4j/issues/16#issuecomment-479820732). Found this problem in few modules, hence their transitive dependencies were not available when building Basic project.  

So in order to fix this problem dependencies are moved from Maven profiles to dependencies section. This was done for following modules: 
- cxf-client-ws
- cxf-server-ws
- jpa-basic
- web

### CODE REVIEW

**Comment by AbhayChandel** [comment]
I am not sure why the _enterprise_ project do not fail due to this issue if the archetype is same for both.

But that do suggest that _Basic_ project has dependency on modules that are fixed here whereas _Enterprise_ does not.

**Comment by AbhayChandel** [comment]
Have tested this with Java 8 and looks fine. @hohwille let me know what do you think how to proceed from here.

File: modules/web/pom.xml [hunk 1]
```diff
@@ -1,49 +1,42 @@
...
-  <version>${devon4j.version}</version>
-  <packaging>jar</packaging>
-  <name>${project.artifactId}</name>
-  <description>Module for reusable web and servlet related code of the Open Application Standard Platform for Java (devon4j).</description>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.devonfw.java.dev</groupId>
```
Comment (hohwille) [inline modules/web/pom.xml:7]: something is wrong with your formatter / indendation.
However, this is a nice-to-have that should not prevent the PR being merged.
Anyhow, please check your IDE settings and fix this for further work to make diffs more readable.

**Review by hohwille — APPROVED** [review]
@AbhayChandel thanks for tracing down this issue and fixing it.
If I got it right the profile error we found in maven seems to apply in general so you had to resolve it in all places.
I will merge this and we can again verify if then the official travis build in devon4j PR gets green.

**Pull request merged**